CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+@Alympic
```

Comment:
https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

i believe we have to add a * in front. can you double check?

Reply:
from example, without *

`# In this example, @octocat owns any file in an apps directory
# anywhere in your repository.
apps/ @octocat

# In this example, @doctocat owns any file in the `/docs`
# directory in the root of your repository and any of its
# subdirectories.
/docs/ @doctocat`